pkg/minikube/bootstrapper/bsutil/featuregates_test.go

```diff
@@ -30,18 +30,18 @@ func TestParseFeatureArgs(t *testing.T) {
 		expectedComponentFeatureArgs string
 	}{
 		{
-			description:  "CoreDNS enabled",
-			featureGates: "CoreDNS=true",
```

Comment:
could you specify why this change is realted to this PR ? is this needed ?

Reply:
v1.18.0 => has updated the k8s.io/kubernetes/cmd/kubeadm/app/features as below : 
var coreDNSMessage = "featureGates:CoreDNS has been removed in v1.13\n" +
	"\tUse kubeadm-config to select which DNS addon to install."

```
const (
	// IPv6DualStack is expected to be alpha in v1.16
	IPv6DualStack = "IPv6DualStack"
	// PublicKeysECDSA is expected to be alpha in v1.19
	PublicKeysECDSA = "PublicKeysECDSA"
)

// InitFeatureGates are the default feature gates for the init command
var InitFeatureGates = FeatureList{
	IPv6DualStack:   {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
	PublicKeysECDSA: {FeatureSpec: featuregate.FeatureSpec{Default: false, PreRelease: featuregate.Alpha}},
}
```

We initialize and validate in ```featuregates.go```. So this update.

Reply:
do you mind pasting what was the exact unit test error you got without this change?

Reply:
sure.
```
--- FAIL: TestParseFeatureArgs (0.00s)
    --- FAIL: TestParseFeatureArgs/CoreDNS_enabled (0.00s)
        featuregates_test.go:59: Kubeadm Actual: map[], Expected: map[CoreDNS:true]
        featuregates_test.go:63: Component Actual: CoreDNS=true, Expected: 
    --- FAIL: TestParseFeatureArgs/CoreDNS_disabled (0.00s)
        featuregates_test.go:59: Kubeadm Actual: map[], Expected: map[CoreDNS:false]
        featuregates_test.go:63: Component Actual: CoreDNS=false, Expected: 
--- FAIL: TestSupport (0.00s)
    featuregates_test.go:90: expected supportedFG(CoreDNS) to be true ! 
```

Reply:
Hey @medyagh - would you like to point out anything specific regarding these changes ?